cmake/Dependencies.cmake

```diff
@@ -351,8 +351,8 @@ if(BUILD_PYTHON)
     list(APPEND Python_ADDITIONAL_VERSIONS "${_py_version}")
   endif()
   list(APPEND Python_ADDITIONAL_VERSIONS 3.6 3.5 2.8 2.7 2.6)
-  find_package(PythonInterp 2.7)
-  find_package(PythonLibs 2.7)
+  find_package(PythonInterp 3.6)
+  find_package(PythonLibs 3.6)
```

Comment:
I know I've complained about these lines of cmake before. I am going to complain about them some more now. Why are you changing the version here?